core/src/main/java/eu/fasten/core/plugins/KafkaPlugin.java

```diff
@@ -0,0 +1,16 @@
+package eu.fasten.core.plugins;
```

Comment:
* No license header
* This class needs to be thoroughly documented

Reply:
Added license headers to all classes and also documented this interface.